src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,46 @@
 layout: content.html
 ---
 
+<%
+const require = this.process.mainModule.require
+const fs = require('fs')
+%>
+
 # Plugins
-*Seneca* plugins enable you to build awesome microservices, incredibly fast.
+
+_Seneca_ plugins enable you to build awesome microservices, incredibly fast.
 Some plugins are maintained in-house, while others are provided by the community.
 
 On this page, we provide an `npm` link and total downloads per month for any plugins
 we find or publish ourselves, so you only need to look in one place.
 
+<!-- <%
+let testFile = JSON.parse(fs.readFileSync('./assets/files/plugins.json', 'utf8'))
+%>
+<%
+testFile.forEach(function (group) {
+  let plugins = Object.values(group)[0]
+  plugins.forEach(function (plugin) {
+%>
+### [<%=plugin.title%>](<%=plugin.title%>)
```

Comment:
the title is human readable, thus not a good choice for a unique "code" - maybe use the plugin name?

Reply:
With the basic handwritten data, plugin.name and plugin/title are the same string. Could change as the real data is pulled in (ex title = seneca-gateway, name = @seneca/gateway from package.json)